web/client/utils/LayersUtils.js

```diff
@@ -385,7 +385,7 @@ export const getLayersByGroup = (configLayers, configGroups) => {
                 group.nodes = getLayersId(groupId, mapLayers).concat(group.nodes)
                     .reduce((arr, cur) => {
                         isObject(cur)
-                            ? arr.push({node: cur, order: mapLayers.find((el) => el.group === cur.id).storeIndex})
+                            ? arr.push({node: cur, order: mapLayers.find((el) => el.group === cur.id)?.storeIndex})
                             : arr.push({node: cur, order: mapLayers.find((el) => el.id === cur).storeIndex});
```

Comment:
what about accessing storeIndex in the false case?

Reply:
False case will be triggered when "cur" is plain ID, which is the case for layer. Each available layer will be inside mapLayers array. But I agree, to be solid sure, let's add conditional chaining in false condition too